dropshot_endpoint/src/api_trait.rs

```diff
@@ -1182,22 +1251,165 @@ impl<'ast> ApiEndpoint<'ast> {
     }
 }
 
+fn parse_channel_metadata(
+    name_str: &str,
+    attr: &syn::Attribute,
+    errors: &ErrorSink<'_, Error>,
+) -> Option<ValidatedChannelMetadata> {
+    // Attempt to parse the metadata -- it must be a list.
+    let l = match &attr.meta {
+        syn::Meta::List(l) => l,
+        _ => {
+            errors.push(Error::new_spanned(
+                &attr,
+                format!(
+                    "endpoint `{name_str}` must be of the form \
+                     #[channel {{ protocol = WEBSOCKETS, path = \"/path\", ... }}]"
+                ),
+            ));
+            return None;
+        }
+    };
+
+    // TODO: Switch to from_tokenstream_spanned once
+    // https://github.com/oxidecomputer/serde_tokenstream/pull/194 is available.
```

Comment:
I think it is, right?

Reply:
Yeah, I missed removing this. Will do, thanks!